packages/cubejs-schema-compiler/src/compiler/CubeSymbols.ts

```diff
@@ -1002,7 +1003,7 @@ export class CubeSymbols {
           cubeName,
           name
         );
-        if (resolvedSymbol._objectWithResolvedProperties) {
+        if (resolvedSymbol && resolvedSymbol._objectWithResolvedProperties) {
```

Comment:
This one is the one that made the error about undefined not having `_objectWithResolvedProperties` property. I kept the other one since it also seems needed. If we don't add this validation, we will reencounter the error when trying to access a member that should exist but doesn't. I'm not sure how to add a better way to validate that.